src/scenarios/emptyvbconsoletemplate/post.py

```diff
@@ -2,10 +2,6 @@
 post cleanup script
 '''
 
-from performance.common import remove_directory
-from shared.const import BINDIR, PUBDIR, APPDIR, TRACEDIR
- 
-remove_directory(BINDIR)
-remove_directory(PUBDIR)
-remove_directory(APPDIR)
-remove_directory(TRACEDIR)
+from shared.postcommands import clean_directories
+
+clean_directories()
```

Comment:
Aren't we changing the semantics here? Before we did not delete the TMPDIR, but now we are. My assumption is that these tests never used TMPDIR so this is unlikely to be a problem. remove_directory works fine if they directory does not exists, right?

Reply:
yes the nice part of remove_directory is it silently ignores the directories that don't exist